packages/worker/src/worker-options.ts

```diff
@@ -704,9 +675,18 @@ export function addDefaultWorkerOptions(options: WorkerOptions): WorkerOptionsWi
     showStackTraceSources: showStackTraceSources ?? false,
     reuseV8Context: reuseV8Context ?? false,
     debugMode: debugMode ?? false,
-    interceptors: appendDefaultInterceptors({}),
+    interceptors: {
+      activityInbound: [
+        ...(options.interceptors?.activityInbound ?? []),
+        (ctx) => new ActivityInboundLogInterceptor(ctx),
+      ],
+      workflowModules: options.interceptors?.workflowModules ?? [],
+    },
     nonStickyToStickyPollRatio: nonStickyToStickyPollRatio ?? 0.2,
-    sinks: { ...defaultSinks(), ...sinks },
+    sinks: {
+      ...initLoggerSink(),
```

Comment:
Wondering if it's worth initializing this with the runtime's sink here to avoid obtaining it later from the runtime instance.